api.graphql

```diff
@@ -22,3 +25,40 @@ type PluginReturn {
 	plugin: String!
 	errors: [Status!]!
 }
+
+# Update this with every plugin type!
+type PluginMasterReturn  {
+	live_site: LiveSiteConfigType
+	slack: SlackConfigType
+	twilio: TwilioConfigType
+	twitter: TwitterConfigType
+}
+
+type MessageReturn {
+    message: String
+    config: MetaDataType!
+    _id: ID!
+    createdAt: String!
+    errors: [Status!]!
+}
+
+type LiveSiteConfigType {
```

Comment:
I meant for you to remove these, see what's done to generate `processedPluginTypeDefs` in `typeDefs.ts` (.schema func per plugin, instead of defining ConfigType per plugin)